<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapViewOpt</class>
 <widget class="QDialog" name="MapViewOpt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Map View Options</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="9" column="0" colspan="2">
    <widget class="QLineEdit" name="EditApiKey"/>
   </item>
   <item row="5" column="0">
    <widget class="QLineEdit" name="MapTitle4"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Title</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLineEdit" name="MapTitle1"/>
   </item>
   <item row="3" column="0">
    <widget class="QLineEdit" name="MapTitle2"/>
   </item>
   <item row="6" column="0">
    <widget class="QLineEdit" name="MapTitle5"/>
   </item>
   <item row="10" column="0" colspan="2">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="BtnOk">
        <property name="text">
         <string>Ok</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BtnCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Google Maps API Key</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLineEdit" name="MapTitle3"/>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Map Tile URL</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="MapTile1"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Leaflet Map Tiles</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLineEdit" name="MapTitle6"/>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="MapTile2"/>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="MapTile3"/>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="MapTile4"/>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="MapTile5"/>
   </item>
   <item row="7" column="1">
    <widget class="QLineEdit" name="MapTile6"/>
   </item>
   <item row="8" column="1">
    <widget class="QPushButton" name="BtnNotes">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>?</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
